index.php

```diff
@@ -37,8 +37,8 @@
 */
 
 // Check PHP version not to have trouble
-if (version_compare(PHP_VERSION, "5.4.0") < 0) {
-   die("PHP >= 5.4.0 required");
+if (version_compare(PHP_VERSION, GLPI_MIN_PHP) < 0) {
+   die(sprintf("PHP >= %s required", GLPI_MIN_PHP));
 }
```

Comment:
Ohhh... Good catch. I wonder if this is not the time to split some of constants in another file (and out of config !) ; the point with the constant is to avoid changing manually the version (and the translation) in several places.